pkg/runtime/subscription/subscription.go

```diff
@@ -162,7 +169,9 @@ func New(opts Options) (*Subscription, error) {
 				return err
 			}
 		} else {
+			// all messages consumed with "rawPayload=false" are deserialized as a CloudEvent, even when the payload is not a CloudEvent
```

Comment:
Is it documented?

Reply:
What would be the best place to document it?

Reply:
this is an implementation detail tbh. It doesn't mean that they will be sent as CloudEvent. it's just that they are being deserialized as cloud events regardless of whether they are rawPayload = true or false.